docs/src/transaction_fees.md

```diff
@@ -1,21 +1,70 @@
 ---
 title: Transaction Fees
+description: "Transaction fees are the small fees paid to process instructions on the network. These fees are based on computation and an optional prioritization fee."
+keywords: "instruction fee, processing fee, storage fee, low fee blockchain, gas, gwei, cheap network, affordable blockchain"
 ---
 
-**Subject to change.**
+The small fees paid to process [instructions](./terminology.md#instruction) on the Solana blockchain are known as "_transaction fees_".
 
-Each transaction sent through the network, to be processed by the current leader validation-client and confirmed as a global state transaction, contains a transaction fee. Transaction fees offer many benefits in the Solana economic design, for example they:
+As each transaction (aka collection of instructions) is sent through the network, it gets processed by the current leader validation-client. Once confirmed as a global state transaction, this _transaction fee_ is paid to the network to help support the [economic design](#economic-design) of the Solana blockchain.
 
-- provide unit compensation to the validator network for the CPU/GPU resources necessary to process the state transaction,
+> **NOTE:** Transaction fees are different from [account rent](./terminology.md#rent)!
+> While transaction fees are paid to process instructions on the Solana network, rent is paid to store data on the blockchain.
+
+<!-- > You can learn more about rent here: [What is rent?](./developing/intro/rent.md) -->
```

Comment:
Let's cleanup comments

Reply:
To get rid of it? I left it in there since when that "what is rent" page gets added we can easily link back to it in the future. The page is written in the WIP PR #26816 but is waiting on the sidebar restructure to get complete to be published

If you want, I can remove it for now though :)

Reply:
We'll remove later when the rent page is added